lib/CMakeLists.txt

```diff
@@ -271,6 +271,7 @@ target_include_directories(arango PUBLIC ${BUILD_DIR})
 
 target_include_directories(arango SYSTEM PUBLIC ${OPENSSL_INCLUDE_DIR})
 target_include_directories(arango SYSTEM PUBLIC ${ICU_INCLUDE_DIR})
+target_include_directories(arango SYSTEM PUBLIC ${Lz4_INCLUDE_DIR})
```

Comment:
LZ4 is now needed for compression/decompression of requests in the SimpleHttpClient module and lib/Basics/EncodingUtils.*